docs/pr-review-guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Postman PR review guidelines
+
+This document sets baseline standard for our PR reviews and serves as a mechanism to align individual postmangineer's expectations of a PR review, both from a reviewer's and am author's point of view.
+
+## Principles
+
+- Technical facts and data overrule opinions and personal preferences.
+- Decisions should **always** been made through either solid engineering principles, and/or through empirical data. If there're multiple equally valid alternatives, reviewers should accept the author's preference.
```

Comment:
> - ****Award all ties to your reviewer**** - Some decisions are just subjective. Award ties to your reviewer since they would have a better perspective on readability (as they are not the author).

I came [across this](https://www.notion.so/opengov/Lesson-2-Implementing-the-feature-883476cec33d452d8ee5f54d9eb17213?pvs=4#79357ecf2ffe4f199f6e997d3e139af4) when I was going through HealthTech's eng docs, which is the opposite of `If there're multiple equally valid alternatives, reviewers should accept the author's preference` and which I think makes more sense hmm

Reply:
I think they might be talking about a different matter. `Some decisions are just subjective` is not the same as `equally valid alternatives` - those could include styling and code structure as well. Also, if reviewer `has a better perspective on readability`, then the person should be able to spell it out and make these alternatives not equal anymore.